app/templates/_gitignore

```diff
@@ -5,6 +5,7 @@ ui/fonts
 ui/styles/*.css
 deploy/local.properties
 local.json
+ecosystem.json
```

Comment:
Hmm I think if it's repository safe devs can put the deploy target in boot.json. It would also just be fairly useful for people to have somewhere they can put their user specific info that's safe from git, just in case they need to put in credentials.